lib/paratii.core.users.js

```diff
@@ -12,4 +12,16 @@ export class ParatiiCoreUsers {
     let options = dopts(config, defaults, {allowUnknown: true})
     this.config = options
   }
+
+  create (options) {
+    return this.paratii.eth.users.create(options)
+  }
+
+  get (id) {
+    return this.paratii.eth.users.get(id)
```

Comment:
It is slightly more complex - if yo look at `paratii.core.vids` you see the pattern to use. 

- data is stored partly on the blockchain, and partly in ipfs, so core needs to decide which goes where
- the data is then idnexed for searching, so `get` should map to paratii.db.vids.get.